### DISCUSSION TITLE
有国内的CDN地址吗

### DISCUSSION THREAD

**Discussion by xmsz** [DouyinFE/semi-design#1958] [Q&A]
unpkg这个基本上经常被墙... 或者特别慢
有没有国内的版本 比如alicdn这种自己的

**Comment by shijiatongxue** [comment]
可以用 [npmmirror 镜像站](https://link.zhihu.com/?target=https%3A//npmmirror.com/) （前身也常被称呼为 淘宝 npm 源）。

- semi-ui.min.js：https://registry.npmmirror.com/@douyinfe/semi-ui/latest/files/dist/umd/semi-ui.min.js
- semi.min.css：https://registry.npmmirror.com/@douyinfe/semi-ui/latest/files/dist/css/semi.min.css

其他文件、使用详情参考：https://zhuanlan.zhihu.com/p/633904268
